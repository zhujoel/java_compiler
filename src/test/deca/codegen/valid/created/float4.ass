; start main program
; Class Declaration
; Code de la table des m√©thodes de Object
	LOAD #null, R0
	STORE R0, 8(GB)
	LOAD code.Object.equals, R0
	STORE R0, 9(GB)
; Main Function
; Test pour savoir si la pile est pleine
	TSTO #3
	BOV pile_pleine
	ADDSP #3
; Declaration des variables
	LOAD #0x1.0p0, R2
	STORE R2, 10(GB)
; Beginning of main instructions:
; /
	LOAD 10(GB), R2
	LOAD #0x1.b33334p1, R3
	DIV R2, R3
	STORE R3, 10(GB)
	LOAD 10(GB), R1
	WFLOAT
	HALT
; end main program
