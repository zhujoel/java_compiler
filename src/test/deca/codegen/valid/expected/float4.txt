2.94118e-01
